packages/modules/src/networkAnalysisDataVis/components/NetworkAnalysisVisualization.styles.ts

```diff
@@ -1,6 +1,28 @@
 import { makeStyles, Theme } from '@material-ui/core/styles'
 
-export default makeStyles((theme: Theme) => ({
+/**
+ * TODO: this is already declared in `client-ui`. Consider
```

Comment:
I suppose we could/should do that.

Reply:
I thought moving it here would be a bit too much in terms of changes. I'll do this as a separate PR.

Reply:
Yeah, that's probably a good idea. :)